tests/ui/specs/SwitchItem/SwitchItem-specs.js

```diff
@@ -23,12 +23,12 @@ describe.skip('SwitchItem', function () {
 		});
 
 		describe('5-way', function () {
-			it('should select the item when selected - [GT-21377]', function () {
+			it('should select the item when selected', function () {
```

Comment:
Would it be possible to match TC GT-21377 with TC QWT-2527? 
```suggestion
			it('should select the item when selected - [QWT-2527]', function () {
```

Reply:
As the QWT-2527 test is the pointer mode test, and this test is a 5-way test, I don't think this test and the test of QWT-2527 are the same. I think the test on 50 line is the same as QWT-2527, what do you think?

Reply:
Sorry, I forgot to check the test mode:( I think we can just leave it as you fixed it.